lib/game_teams_manager.rb

```diff
@@ -59,4 +59,117 @@ def add_wins_losses(gt_results, coach_record_start)
     end
     coach_record_start
   end
+
+#-------------TeamStats
+  def best_season(team_id)
+    best_season = win_percentage_by_season(team_id).max_by do |season, wins_percent|
+        wins_percent
+      end
+      best_year = best_season[0].to_i
+      "#{best_year}201#{best_year.digits[0] + 1}"
+    end
```

Comment:
The spacing on these `ends` looks like is a space off

Reply:
Updated, thanks for finding that!